README.md

```diff
@@ -102,18 +101,20 @@ $ kubectl exec -it samplepod -- ip a
    <img src="doc/images/multus_cni_pod.png" width="1008" />
 </p>
 
-## Build
+## Building from source
 
-**This plugin requires Go 1.8 to build.**
+**This plugin requires Go 1.8 (or later) to build.**
 
 Go 1.5 users will need to set GO15VENDOREXPERIMENT=1 to get vendored dependencies. This flag is set by default in 1.6.
```

Comment:
We can remove "GO15VENDOREXPERIMENT=1" because we said multus requires golang >=1.8.

Reply:
Good eye! Pushing up a tweak.